Networking/NetworkingTests/Remote/StripeRemoteTests.swift

```diff
@@ -305,8 +305,8 @@ final class StripeRemoteTests: XCTestCase {
         // Then
         XCTAssertTrue(result.isSuccess)
         let account = try result.get()
-        XCTAssertEqual(account.isLiveAccount, true)
-        XCTAssertEqual(account.isInTestMode, false)
+        XCTAssertEqual(account.isLiveAccount, false)
```

Comment:
This will have to be reverted when we remove the hardcoded properties in the StripeAccount object, after the extension is updated